### PR TITLE
Actionable Intelligence scoring

### PR DESCRIPTION
**Pull request by prattmic**
Score objective/threshold level of actionable intelligence and include
the result in the evaluation dict. No scores are used at this point,
simply an objective/threshold string.

sample_mission.json is unchanged expect for the addition of real and
submitted targets. Unfortunately the database dump reordered the file.

References #31

### CODE REVIEW

File: server/auvsi_suas/models/target.py [hunk 1]
```diff
@@ -268,6 +269,25 @@ def similar_classifications(self, other):
...
+        """Checks if Target meets Actionable Intelligence submission criteria.
+
+        A target is "actionable" if the aircraft was in flight from initial
+        target submission until the last update. Note that this does not check
+        the localization or characterization Actionable Intelligence criteria.
+
+        Returns:
+            True if target may be considered an "actionable" submission.
+        """
+        flights = TakeoffOrLandingEvent.flights(self.user)
```
Comment (pmtischler) [inline server/auvsi_suas/models/target.py]: Do we want to pass this in and drop it from `O(targets)` queries to `O(1)`?

**Comment by pmtischler** [comment]
Overall LGTM. Note that we may need to change the match value to include this info based on our offline email thread.

**Comment by pmtischler** [comment]
Per the issue thread, let's add it to the match value and use 2,1 as point values (in settings file) for the scoring.

File: server/auvsi_suas/models/target.py (hunk 1 above)
Comment (prattmic) [inline server/auvsi_suas/models/target.py]: Done

**Comment by prattmic** [comment]
Done. PTAL

File: server/auvsi_suas/models/target_test.py [hunk 2]
```diff
@@ -327,22 +377,29 @@ def setUp(self):
...
         """Tests the match value for two targets."""
-        e = TargetEvaluator([], [])
-        self.assertEqual(6, e.match_value(self.submit1, self.real1))
+        e = TargetEvaluator(self.submitted_targets, self.real_targets)
+        self.assertEqual(8, e.match_value(self.submit1, self.real1))
         self.assertEqual(3, e.match_value(self.submit2, self.real2))
         self.assertEqual(0, e.match_value(self.submit3, self.real3))
         self.assertEqual(0, e.match_value(self.submit4, self.real4))
 
     def test_match_targets(self):
```
Comment (pmtischler) [inline server/auvsi_suas/models/target_test.py]: Please add a test case for where real is an empty list, or submitted is an empty list.

File: server/server/settings.py [hunk 3]
```diff
@@ -209,3 +209,17 @@
      "end": float("inf"),
      "value": 0},
 ]
+
+# Actionable Intelligence threshold and objective level parameters.
+TARGET_ACTIONABLE_PARAMS = {
+    "threshold": {
+        "characteristics": 3. / 5.,
```
Comment (pmtischler) [inline server/server/settings.py]: Would it be better to be semantic instead (say threshold level actionable requires threshold level characteristics & location), rather than via value? That way the two will stay in sync. It may require restructuring the original structure to use named values.

**Comment by pmtischler** [comment]
LGTM after comments.

File: server/server/settings.py (hunk 3 above)
Comment (prattmic) [inline server/server/settings.py]: Imagery threshold requires only two characteristics, but Actionable intelligence threshold requires three characteristics.

Comment (pmtischler) [inline server/server/settings.py]: :0 good catch. Sounds good to do it this way.

File: server/auvsi_suas/models/target_test.py (hunk 2 above)
Comment (prattmic) [inline server/auvsi_suas/models/target_test.py]: Done
